ocs_ci/utility/users.py

```diff
@@ -64,6 +64,41 @@ def delete_htpasswd_secret():
     exec_cmd(cmd)
 
 
+def delete_user(user_name=None):
+    """
+    Delete user
+
+    Args:
+        user_name (str): the user to delete
+    """
+    if user_name is None:
+        cmd = "oc delete user --all"
```

Comment:
Please update the docstring to reflect that all users will be deleted of no user name is provided or divide the function into two: `delete_user`  with mandatory parameter and `delete_all_users`.

Reply:
done